docs/reference/manifest/runtime.md

```diff
@@ -47,7 +47,17 @@ For more information, see [Version overrides in the manifest](../../develop/add-
 |  Attribute  |  Required  |  Description  |
 |:-----|:-----|:-----|
 |  **resid**  |  Yes  | Specifies the URL location of the HTML page for your add-in. The `resid` can be no more than 32 characters and must match an `id` attribute of a `Url` element in the `Resources` element. |
-|  **lifetime**  |  No  | The default value for `lifetime` is `short` and doesn't need to be specified. Outlook add-ins use only the `short` value. If you want to use a shared runtime in an Excel add-in, explicitly set the value to `long`. |
+|  [lifetime](#lifetime-attribute)  |  No  | The default value for `lifetime` is `short` and doesn't need to be specified. Outlook add-ins use only the `short` value. If you want to use a shared runtime in an Excel add-in, explicitly set the value to `long`. |
+
+### lifetime attribute
+
+Optional. Represents the length of time the add-in is allowed to run.
+
+**Available values**
+
+`short`: Default. After the add-in is activated, it will be allowed to run for a maximum amount of time as specified by the platform. Currently, that's around 5 minutes. This is the only value supported by Outlook.
```

Comment:
```suggestion
`short`: Default. Used only for Outlook add-ins. After the add-in is activated, it will run for a maximum amount of time as specified by the platform. Currently, that's around 5 minutes. This is the only value supported by Outlook.
```